Document on the Order Lookup feature

AjinkyaM1 opened the pull request:
The document explains the need for this screen and outlines the various types of information it provides within the Fulfillment App.

AjinkyaM1 committed:
Document on the Order Lookup screen

The document explains the need for this screen and outlines the various types of information it provides within the Fulfillment App.

AjinkyaM1 committed:
Added summary for the Order Lookup page

AjinkyaM1 committed:
Corrected the typing mistake in the summary

dt2patel commented on adoc/flows/SalesOrders/order-lookup.md:
```diff
@@ -0,0 +1,71 @@
+---
+description: >-
+  The document highlights the HotWax Commerce Order Lookup feature, which enables store associates to efficiently manage in-store order inquiries with direct access to comprehensive order information.
```
```suggestion
  The document highlights the HotWax Commerce Order Lookup feature, which enables store associates to manage in-store order inquiries with direct access to comprehensive order information.
```

AjinkyaM1 replied:
Removed the order approval changes

AjinkyaM1 committed:
Updated the ADOC approval process documentation

The document is revised with the latest updates in the approval process in ADOC

AjinkyaM1 committed:
Changed the document with the latest requirements

Updated the document to make it more specific to ADOC, rather than keeping it as the previously written generic version

dt2patel commented on adoc/flows/SalesOrders/OrderApproval.md:
```diff
@@ -5,83 +5,42 @@ description: >-
 ---
 
-# Order Approval
+# Order Approval Process in ADOC
```
```suggestion
# Order Approval Process
```

dt2patel reviewed (changes requested):
Please remove Order Approval related changes from this PR.

AjinkyaM1 committed:
Merge branch 'hotwax:implementations-pub' into implementations-pub

AjinkyaM1 committed:
Reverted the changes regarding order approval

Reverted the changes related to order approval process
